Champions/Nidalee/R.cs

```diff
@@ -0,0 +1,88 @@
+using GameServerCore.Domain.GameObjects;
+using static LeagueSandbox.GameServer.API.ApiFunctionManager;
+using LeagueSandbox.GameServer.GameObjects.AttackableUnits.AI;
+using GameServerCore.Domain;
+using LeagueSandbox.GameServer.Scripting.CSharp;
+
+namespace Spells
+{
+    public class AspectOfTheCougar : IGameScript
+    {
+        static ISpell spell;
+        public void OnActivate(IChampion owner)
+        {
+            //TODO:when game started, set level as 1;
+            
+        }
+
+        public void OnDeactivate(IChampion owner)
+        {
+        }
+
+        public void OnStartCasting(IChampion owner, ISpell spell, IAttackableUnit target)
+        {
+            var p1 = AddParticleTarget(owner, "Nidalee_Base_R_Cas.troy", owner, 1);
+            if (owner.Model == "Nidalee_Cougar")
+            {               
+                var a = owner.Spells[0].Level;
+                var b = owner.Spells[1].Level;
+                var c = owner.Spells[2].Level;
+                owner.ChangeModel("Nidalee");
+                owner.Stats.Range.FlatBonus += 400;
+                owner.SetSpell("JavelinToss", 0, true);
+                owner.SetSpell("Bushwhack", 1, true);
+                owner.SetSpell("PrimalSurge", 2,true);
+                               
+                for (byte p = 1; p <= a; p++)
+                {
+                    owner.Spells[0].LevelUp();
+                }
+                for (byte p = 1; p <= b; p++)
+                {
+                    owner.Spells[0].LevelUp();
+                }
+                for (byte p = 1; p <= c; p++)
+                {
+                    owner.Spells[2].LevelUp();                    
+                }
+                return;
+            }
+            if (owner.Model == "Nidalee")

```

Comment:
Many hardcoded constants.

Reply:
well, because when I changed Model, and Q,W,E were disabled, So I use like that
1.detect which model that owner having
2. make spells enable
3.get the previous level, then level up them